sys-block/tcmu-runner/tcmu-runner-1.4.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# with reference to:
+#	ebuild: sys-block/open-iscsi
+#	ebuild: sys-block/fio
+#	ebuild: app-admin/conky
+#	tcmu-runner.spec
+#	https://devmanual.gentoo.org/ebuild-writing/eapi/index.html
+#
+
+EAPI=6
+
+inherit cmake-utils linux-info
+
+DESCRIPTION="A daemon that handles the userspace side of the LIO TCM-User backstore"
+HOMEPAGE="http://www.open-iscsi.com/"
+SRC_URI="https://github.com/open-iscsi/${PN}/archive/v${PVR}.tar.gz -> ${PF}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="rbd glfs +zbc +qcow +systemd"
+
+DEPEND="
+	dev-libs/glib:2
+	dev-libs/libnl:3
+	sys-libs/zlib
+	sys-apps/kmod
```

Comment:
Same here. You've managed to do unconditionals before conditionals right, but you can sort unconditionals alphabetically by category, and conditionals alphabetically by their USE flags.